### ISSUE TITLE
Facial cutaneous polyps

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11590]
**Preferred term label:**
Facial cutaneous polyps

**Synonyms**

**Definition (free text, please give PubMed ID)**
Fetal facial skin projections, imaged by prenatal ultrasound

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pai syndrome OMIM % 155145
encephalocraniocutaneous lipomatosis OMIM # 613001
Sakoda complex MIM 610871

**Your nano-attribution (ORCID)**
0000-0001-9947-035X
